### PR TITLE
app-i18n/ibus-anthy: Version bump to 1.5.9

### PR DESCRIPTION
**Pull request by tietzc**
Gentoo-Bug: https://bugs.gentoo.org/597580

ibus-anthy-1.5.4 in the tree is ancient. Therefore, despite not proxy-maintaing this, I would like to bump it to the lastest upstream version. Optionally cleaning up googlecode references, too.

Build log: https://bpaste.net/show/80e7dffdd233

### CODE REVIEW

File: app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+src_configure() {
+	econf \
+		--libexecdir=/usr/libexec \
+		--enable-private-png \
+		$(use_enable nls)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+	dodoc AUTHORS ChangeLog NEWS README
```
Comment (SoapGentoo) [inline app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild]: remove the previous two lines, and just use `default`

File: app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools eutils gnome2-utils python-single-r1
```
Comment (SoapGentoo) [inline app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild]: you can likely also remove `eutils`

File: app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+DEPEND="${RDEPEND}
+	dev-libs/gobject-introspection
+	dev-util/intltool
+	nls? ( sys-devel/gettext )
+	virtual/pkgconfig"
+
+src_prepare() {
+	default
+	eautoreconf
+	>py-compile #397497
```
Comment (mgorny) [inline app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild]: I'm 99% sure this is wrong / a relict of old python.eclass.

File: app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+src_configure() {
+	econf \
+		--libexecdir=/usr/libexec \
```
Comment (mgorny) [inline app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild]: Do you actually need this? It looks like the default to me. Besides, wouldn't harm to prepend `"${EPREFIX}"` to the path.

File: app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+		--enable-private-png \
+		$(use_enable nls)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+	dodoc AUTHORS ChangeLog NEWS README
+	find "${ED}" -name '*.la' -type f -delete || die
+	python_optimize
```
Comment (mgorny) [inline app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild]: Once you restore `py-compile` to normal, please check if this is still necessary.

**Comment by tietzc** [comment]
Thanks for the input everyone. All suggestions have proven applicable.

New build log: https://bpaste.net/show/237145f781eb

File: app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+RDEPEND="${PYTHON_DEPS}
+	app-i18n/anthy
+	app-i18n/ibus[introspection]
+	nls? ( virtual/libintl:0= )"
+
+DEPEND="${RDEPEND}
+	dev-libs/gobject-introspection
+	dev-util/intltool
+	nls? ( sys-devel/gettext )
```
Comment (mgorny) [inline app-i18n/ibus-anthy/ibus-anthy-1.5.9.ebuild]: Minor nit: we usually put USE conditionals after unconditional deps.

**Comment by tietzc** [comment]
Put USE conditional last in DEPEND.

**Comment by mgorny** [comment]
@SoapGentoo, please update your review

**Comment by mgorny** [comment]
@gentoo/cjk, could you please take a look at this?

**Comment by tietzc** [comment]
One QA check seems to be stuck for more than 24 hours now. Should I rebase against master and push again or is that just irrelevant?

**Comment by mgorny** [comment]
@tietzc, it probably got killed by a system reboot. Please rebase indeed.

**Comment by tietzc** [comment]
@monsieurp, thank you.

**Pull request closed without merging**